test/extensions/common/aws/credentials_provider_impl_test.cc

```diff
@@ -13,9 +14,7 @@
 #include "test/test_common/simulated_time_system.h"
 #include "test/test_common/test_runtime.h"
 
-using Envoy::Extensions::Common::Aws::MetadataFetcher;
 using Envoy::Extensions::Common::Aws::MetadataFetcherPtr;
-using Envoy::Extensions::Common::Aws::MockMetadataFetcher;
```

Comment:
are these unused?

Reply:
this was flagging as follows:
`Using decl 'MockMetadataFetcher' is unused (fix available)`

Envoy::Extensions::Common::Aws::MockMetadataFetcher class is already defined via 

#include "test/extensions/common/aws/mocks.h"